docker_push.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
-DREDD_VERSION=`./scripts/getLatestStableNodePackage dredd`
+DREDD_VERSION=$(npm view dredd version)
```

Comment:
nice!

Reply:
@abtris If you wanted to use the `dist-tags`, it looks like this for `stable`:

```shell
npm view dredd dist-tags --json | grep 'stable' | python -c 'import sys, re; print(re.search(r": \"([^\"]+)", sys.stdin.read()).group(1))'
```

Or like this for `latest`:

```shell
npm view dredd dist-tags --json | grep 'latest' | python -c 'import sys, re; print(re.search(r": \"([^\"]+)", sys.stdin.read()).group(1))'
```

But that's not necessary for Dredd at the moment as we're not doing anything with the tags. Dredd currently always automatically releases the latest version to both `latest` and `stable`.

Reply:
👍

Reply:
or, without python.

```
npm view dredd dist-tags --json | grep 'stable' | cut -d ":" -f2 | tr -d '"'
```

Reply:
Nice. I'm not that skilled in bash and when I run out of ideas, I use Python 😛